python/cudf/tests/test_groupby.py

```diff
@@ -35,16 +37,47 @@ def get_nelem():
         yield elem
 
 
+@pytest.fixture
+def gdf():
+    return DataFrame({'x': [1, 2], 'y': [1, 2]})
+
+
+@pytest.fixture
+def pdf(gdf):
+    return gdf.to_pandas()
```

Comment:
Are these fixtures not defined in the testing utils file? We should just move them there instead of redefining in multiple files.